fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/InsertUtils.java

```diff
@@ -383,6 +395,9 @@ public static TableIf getTargetTable(Plan plan, ConnectContext ctx) {
 
     private static NamedExpression generateDefaultExpression(Column column) {
         try {
+            if (column.getGeneratedColumnInfo() != null) {
+                return new DefaultValueSlot();
```

Comment:
u could not return DefaultValueSlot() here, must replace by valid expression. u can try 
```sql
create table t(id int, c1 int as(c2), c2 int) properties('replication_num'='1');
insert into t values(1, default, 2), (2, default, 3);
```